game/scripts/npc/abilities/ember_spirit_flame_guard.txt

```diff
@@ -20,11 +20,11 @@
 
     // Time
     //-------------------------------------------------------------------------------------------------------------
-    "AbilityCooldown"                                     "35.0"
+    "AbilityCooldown"                                     "35.0 32.0 29.0 26.0 23.0 20.0" //OAA
 
     // Cost
     //-------------------------------------------------------------------------------------------------------------
-    "AbilityManaCost"                                     "80 90 100 110 220 350"
+    "AbilityManaCost"                                     "100 110 120 130 230 330" //OAA
```

Comment:
Revert this, and just make lvl 6 mana cost: 330.

Reply:
> Revert this, and just make lvl 6 mana cost: 330.

with the huge buffs I'm giving to flame guard I wanted to make it a little more costly in the early game.

Reply:
and with the fact it's now 20 seconds instead of 8 at level 1 even if with the manacost increase you would still be effectively using more mana before.

Reply:
Ok, seems fair.